Scoop.js

```diff
@@ -948,14 +949,19 @@ export class Scoop {
           const body = await readFile(`${this.captureTmpFolderPath}${file}`)
           const isEntryPoint = false // TODO: Reconsider whether this should be an entry point.
 
-          this.addGeneratedExchange(url, httpHeaders, body, isEntryPoint)
-          videoSaved = true
+          if (body.length) {
```

Comment:
How would you feel about checking for `if !body.length` and then `continue`ing, instead of setting `videoSaved` and then checking it?

Reply:
If you do this, and log something there, then I think you can then remove the block I added entirely:
```
   if (videoSaved === false) {
      this.log.warn('yt-dlp reported success (returned 0), but produced no output.')
      return
    }
```

Reply:
Thanks! Yes, let me overhaul this -- I was thinking the way I did it might not be idiomatic or concise.

Reply:
I think I can't take out your block, as that `return` is what prevents the summary from being created.

Reply:
Oh, or maybe I can move it up higher.